config.yaml

```diff
@@ -48,3 +48,9 @@ options:
     default: 1
     description: |
       Number of executors to configure for jenkins master.
+  public_url:
```

Comment:
Please use dashes as separator, like the other config keys in this file:

"public_url" -> "public-url"